.github/workflows/deploy.yml

```diff
@@ -0,0 +1,81 @@
+name: Deploy
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+  workflow_dispatch:
```

Comment:
What does it do exactly here?

Reply:
It gives you a neat button for re-running the workflow:
![image](https://user-images.githubusercontent.com/1443107/226947984-905b4fcc-8114-42fc-a379-5966151384f0.png)